.github/workflows/set_wheel_build_matrix.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Kokkos execution strategy
         id: exec_model
-        run: echo "exec_model=[\"SERIAL\", \"OPENMP\"]" >> $GITHUB_OUTPUT # We may also adopt [THREADS] in later iterations
```

Comment:
Do we want to get rid of serial here?

Reply:
`set_wheel_build_matrix.yml` is only used for the wheels. I think we can build `lightning_qubit` wheels without any Kokkos and `lightning_kokkos` wheels with Kokkos-OPENMP, and additional backends can be tested in test workflows. Does that work?

Reply:
Sure, that's fine by me.